packages/contract-addresses/addresses.json

```diff
@@ -244,5 +244,87 @@
             "fillQuoteTransformer": "0x99356167edba8fbdc36959e3f5d0c43d1ba9c6db",
             "positiveSlippageFeeTransformer": "0x45b3a72221e571017c0f0ec42189e11d149d0ace"
         }
+    },
+    "137": {
+        "erc20Proxy": "0x0000000000000000000000000000000000000000",
+        "erc721Proxy": "0x0000000000000000000000000000000000000000",
+        "zrxToken": "0x0000000000000000000000000000000000000000",
+        "etherToken": "0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270",
+        "exchangeV2": "0x0000000000000000000000000000000000000000",
+        "exchange": "0x0000000000000000000000000000000000000000",
+        "assetProxyOwner": "0x0000000000000000000000000000000000000000",
+        "zeroExGovernor": "0x0000000000000000000000000000000000000000",
+        "forwarder": "0x0000000000000000000000000000000000000000",
+        "coordinatorRegistry": "0x0000000000000000000000000000000000000000",
+        "coordinator": "0x0000000000000000000000000000000000000000",
+        "multiAssetProxy": "0x0000000000000000000000000000000000000000",
+        "staticCallProxy": "0x0000000000000000000000000000000000000000",
+        "erc1155Proxy": "0x0000000000000000000000000000000000000000",
+        "devUtils": "0x0000000000000000000000000000000000000000",
+        "zrxVault": "0x0000000000000000000000000000000000000000",
+        "staking": "0x0000000000000000000000000000000000000000",
+        "stakingProxy": "0x0000000000000000000000000000000000000000",
+        "erc20BridgeProxy": "0x0000000000000000000000000000000000000000",
+        "erc20BridgeSampler": "0x0000000000000000000000000000000000000000",
+        "chaiBridge": "0x0000000000000000000000000000000000000000",
+        "dydxBridge": "0x0000000000000000000000000000000000000000",
+        "godsUnchainedValidator": "0x0000000000000000000000000000000000000000",
+        "broker": "0x0000000000000000000000000000000000000000",
+        "chainlinkStopLimit": "0x0000000000000000000000000000000000000000",
+        "maximumGasPrice": "0x0000000000000000000000000000000000000000",
+        "dexForwarderBridge": "0x0000000000000000000000000000000000000000",
+        "exchangeProxyGovernor": "0x4d3e56c56a55d23fc7aa9a9ffad61631cf7d1ae6",
```

Comment:
so one annoying thing is we usually use etherscan to interact the governor. maybe we should deploy a flattened version of the governor so we can interact with it on vigil

Reply:
nvm  we'll figure it out